src/main/java/frc/robot/commands/EjectCell.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Shooter;
+
+public class EjectCell extends CommandBase {
```

Comment:
What is the difference between the EjectCell command and the Shoot command? It is not entirely clear.